erpnext/payroll/doctype/additional_salary/additional_salary.json

```diff
@@ -7,25 +7,30 @@
  "editable_grid": 1,
  "engine": "InnoDB",
  "field_order": [
+  "employee_details_section",
   "naming_series",
   "employee",
   "employee_name",
-  "salary_component",
-  "type",
-  "amount",
-  "ref_doctype",
-  "ref_docname",
-  "amended_from",
   "column_break_5",
   "company",
   "department",
+  "salary_component_and_amount_section",
+  "salary_component",
   "currency",
+  "amount",
+  "column_break_13",
+  "type",
```

Comment:
`type` (salary component type) field should be below `salary_component` since it is fetched from it.